docs/data-sources/phylop.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: PhyloP
+---
+
+import JSON from './phylop-json.md'
+
+## Overview
+
+PhyloP (phylogenetic p-values) conservation scores are obtained from the [PHAST package] (http://compgen.bscb.cornell.edu/phast/) for multiple alignments of vertebrate genomes to the human genome. For GRCh38, the multiple alignments are against 19 mammals and for GRCh37, it is against 45 vertebrate genomes.
+
+:::info Publication
+Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. **Genome Res. 2005** Aug;15(8):1034-50.  (http://www.genome.org/cgi/doi/10.1101/gr.3715005)
+:::
+
+
+## File Parsing
```

Comment:
"File Parsing" should be "Wigfix File" to be consistent with dbSNP.

Reply:
Done too.